Tools/actions_changelog_rss.py

```diff
@@ -102,23 +101,40 @@ def main():
 
         et = ET.ElementTree(feed)
         with sftp.open(RSS_FILE, "wb") as f:
-            et.write(f, encoding="utf-8", xml_declaration=True)
-
-
-def create_feed(changelog: Any, previous_items: List[ET.Element]) -> Tuple[ET.Element, bool]:
+            et.write(
+                f,
+                encoding="utf-8",
+                xml_declaration=True,
+                # This ensures our stylesheet is loaded
+                doctype="<?xml-stylesheet type='text/xsl' href='./stylesheet.xsl'?>",
```

Comment:
I don't see this parameter in the documentation: https://docs.python.org/3/library/xml.etree.elementtree.html#xml.etree.ElementTree.ElementTree.write

Reply:
It was the reason to replace it with lxml

Reply:
Ah, I missed the change to lxml